components/apimgt/org.wso2.carbon.apimgt.solace/src/main/java/org/wso2/carbon/apimgt/solace/SolaceAdminApis.java

```diff
@@ -141,6 +141,84 @@ public CloseableHttpResponse registerAPI(String organization, String title, Stri
         return null;
     }
 
+    /**
+     * Update API in Solace using AsyncAPI Definition
+     *
+     * @param organization  name of the Organization
+     * @param title         name of the Solace API
+     * @param apiDefinition Async definition of the Solace API
+     * @return CloseableHttpResponse of the PUT call
```

Comment:
Shall we change this as PATCH Call instead of PUT call as we use a patch in the method?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/11001/commits/9c9fe10fbb271f44e424d8a7efc0a76be0fc7b09